common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -61,10 +61,26 @@ def display_name_with_default(course):
             said course.
     """
     # TODO: Consider changing this to use something like xml.sax.saxutils.escape
+    return course.display_name_unescaped.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
I get that this isn't really part of this PR, but this partial escaping should really be fixed to use any proper escaping method.  Of course this goes away when we get to our ultimate goal, but it is broken in the meantime.